### DISCUSSION TITLE
ROS wiki outdated

### DISCUSSION THREAD

**Discussion by knmcguire** [USC-ACTLab/crazyswarm#509] [General]
Hi! I just thought to give this discussion feature a go :) Since this is not really sure if this suggestion is something within this repo or in another place, so assume this is the right place to ask!

I started to get a bit more active in ROS again, but I noticed that the Wiki: http://wiki.ros.org/crazyflie of the crazyflie_ros repo. It is clear that the orginal  github repo is deprecated, but their wiki still points to it. It would be nice to have a more director connection for the googlers among us.

I guess that this needs to be linked to a particular ROS package? Is that possible still within Crazyswarm?

**Comment by whoenig** [comment]
Very good point - I opened an issue for that. I believe that we can update the normal robot wiki (https://robots.ros.org/category/aerial/), but not the page you linked at, because that one is automatically generated from the package index. I am not sure if we can add the crazyswarm to that index, since its a whole workspace, not a stack. In principle this could be fixed, but might not be worth it for the ROS1 version, since it likely would break compatibility in some way.

**Reply by knmcguire** [reply]
Yes indeed the robots.ros.org website is markdown based so we can easily change that on the github repo [under posts](https://github.com/ros-infrastructure/robots.ros.org). From there, a link goes to http://wiki.ros.org/Robots/Crazyflie, which is something that I'm not sure we can change easily. Perhaps you still have access to it as you edited? 

I kind of agree that it is maybe not worth for ROS1 to fix his properly, but maybe the easy fix is just to notify people on these robot pages. On both the robots.org and wiki robot page we can add a note that the crazyflie_ros package has been expired and people should go to the crazyswarm project immediately (if the ROS maintainers are okay with that).

**Reply by whoenig** [reply]
I fixed http://wiki.ros.org/Robots/Crazyflie (might take some time for the official approval to go through).

**Reply by knmcguire** [reply]
ah great! thanks. 

How about the robot.org part? shall we propose a change now with a PR or wait a bit with that until after the ROS2 intergration?

**Reply by whoenig** [reply]
There doesn't seem to be any problem with that website, or am I missing something?

**Reply by knmcguire** [reply]
Ah yeah, not about the deprecated ROS packages, but the links on that page still refer to the old product pages on the deprecated wiki, which should be to the ones on the website. Also maybe we shouldn't even mention Crazyflie 1.0.

I guess we can copy to content on what you have changed on http://wiki.ros.org/Robots/Crazyflie ? I see that the ROS people have already accepted it!

**Comment by knmcguire** [comment]
btw, the new doc for the packages will be here: https://index.ros.org/p/crazyflie/github-whoenig-crazyflie_ros/#melodic

**Comment by knmcguire** [comment]
Just an update. I've proposed a change to  https://robots.ros.org/crazyflie/ by this pull request: https://github.com/ros-infrastructure/robots.ros.org/pull/235 It's a copy of the changes @whoenig did for the wiki plus a video of the research compilation

**Reply by whoenig** [reply]
Awesome - thanks!

**Comment by knmcguire** [comment]
[robot ros website ](https://robots.ros.org/crazyflie/) has been updated 😄 shall we close the issue or is there any other sources we shouldn't forget?

**Reply by whoenig** [reply]
Great - I closed the issue!
